.jenkinsci/bindings.groovy

```diff
@@ -108,7 +110,8 @@ def doAndroidBindings(abiVersion) {
     [currentPath, "\$PLATFORM", abiVersion, "\$BUILD_TYPE_A", sh(script: 'date "+%Y%m%d"', returnStdout: true).trim(), commit.substring(0,6)])
   sh """
     (cd /iroha; git init; git remote add origin https://github.com/hyperledger/iroha.git; \
-    git fetch --depth 1 origin develop; git checkout -t origin/develop)
+    # TODO possible race on consequent pushes to the same branch
```

Comment:
Could you elaborate on that?

Reply:
What if someone pushed two commits to the same branch in a row? It seems to me that for two running CI pipelines, if the first pipeline did not reach this line with git fetch yet, it will also use the latest commit instead of previous one.